requirements.txt

```diff
@@ -2,7 +2,7 @@ urllib3
 imutils
 numpy
 opencv_python
-Pillow
+pillow
```

Comment:
U sure? It really looks like it should be Pillow on linux too
![grafik](https://github.com/fishyboteso/fishyboteso/assets/1882648/d8558390-b9dc-4168-9eb1-d2f11e7b43da)
https://pillow.readthedocs.io/en/stable/installation.html